app/assets/javascripts/hydra-editor/keyboard_support.js

```diff
@@ -1,14 +0,0 @@
-Blacklight.onLoad(function() {
-  $('body').on('keypress', '.multi-text-field', function(event) {
-    var $activeField = $(event.target).parents('.field-wrapper'),
-        $activeFieldControls = $activeField.children('.field-controls'),
-        $addControl = $activeFieldControls.children('.add'),
-        $removeControl = $activeFieldControls.children('.remove');
-    if (event.keyCode == 13) {
-      event.preventDefault();
-      $addControl.click()
-      $removeControl.click()
-    }
-  });
-});
-
```

Comment:
I'm getting mixed results on the enter key and spacebar issue. I'd say "go for it" since it possibly alleviates accidentally deleting metadata depending on where the focus happens to be in the form.